libs/components/lists/src/lib/modules/list-summary/list-summary.component.scss

```diff
@@ -24,12 +24,22 @@
       var(--sky-comp-list_summary-space-inset-left)
   );
   background-color: var(
-    --sky-override-list-summary-background-color,
```

Comment:
Do we need this switch? If so - seems like the viewkeeper portion below should also be switched. But the override being first seems right to me. (Probably has no effect in practice either way since the override would be in default but the custom property is probably never set there - but it feels like the override should be first).